jenkins/Jenkinsfile

```diff
@@ -10,6 +10,11 @@ pipeline {
       choice(name: 'AWS_REGION', choices: ['us-east-1', 'us-west-2'], description: 'AWS Region to deploy to')
       choice(name: 'MAKE_TARGET', choices: ['all', 'tf', 'daac', 'data-persistence', 'cumulus', 'workflows', 'cumulus_v9_2_0_upgrade'], description: 'What section of cumulus to deploy.')
 
+      booleanParam(name: 'DEPLOY_DASHBOARD', defaultValue: true, description: 'Whether or not to deploy the cumulus dashboard')
```

Comment:
The default should probably be `false` for this repo and we should override it in CIRRUS-ASF